Models/categories.models.js

```diff
@@ -52,6 +52,24 @@ exports.fetchReviews = (category) => {
   });
 };
 
+exports.fetchComments = (review_id) => {
+  return db
+    .query(
+      `SELECT comments.* FROM comments 
+    LEFT JOIN reviews ON comments.review_id = reviews.review_id
+    WHERE comments.review_id=$1
+    GROUP BY comments.comment_id;`,
+      [review_id]
+    )
+    .then(({ rows }) => {
+      if (!rows[0]) {
+        return Promise.reject({ status: 404, msg: "review does not exist" });
+      } else {
```

Comment:
In relation to the additional test you'll need, this .length check will not be sufficient to find out if the review exists but just has no comments!